app/assets/stylesheets/campaigner-facing/page-edit.scss

```diff
@@ -496,3 +496,27 @@ body.page-edit-body {
     opacity: 1;
   }
 }
+
+.call-tool {
+  .block {
+    display: block;
```

Comment:
I was using it on divs,  but was overriding a display:inline by bootstrap. It is ugly I agree, I reafactored it so we don't need this anymore.